src/MultiSelect/SelectionList.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import { multiSelectedPropType } from '../common-prop-types.js'
+import { Chip } from '../Chip.js'
+
+const createRemoveHandler = ({ selected, onChange, value, label }) => () => {
+    const filtered = selected.filter(selection => {
+        const matchesLabel = label === selection.label
+        const matchesValue = value === selection.value
+        return !matchesLabel && !matchesValue
+    })
```

Comment:
Seems to be functionality used in multiple places. Coult be extracted into its own function

Reply:
You're right, makes sense to abstract it. Did that here: 3c8bff032b409f8db906b2aee43ce3e7d2b207f1